shodo/aio/lint.py

```diff
@@ -4,16 +4,16 @@
 import aiohttp
 
 from shodo.aio.api import lint_create, lint_result
-from shodo.lint import Lint, LintFailed, LintResult, Message
+from shodo.lint import LintFailed, LintResult, LintStatus, Message
 
 
 async def lint(body: str, is_html: bool = False, profile: Optional[str] = None) -> LintResult:
     async with aiohttp.ClientSession() as session:
         create_res = await lint_create(body, is_html, profile, session)
 
-        status = Lint.STATUS_PROCESSING
+        status = LintStatus.PROCESSING.value
         pause = 0.25
-        while status == Lint.STATUS_PROCESSING:
+        while status == LintStatus.PROCESSING:
```

Comment:
これだと1ループ目で False になって抜けちゃいますね。
```suggestion
        while status == LintStatus.PROCESSING.value:
```
下の 
```python
if status == LintStatus.FAILED.value:
```
のところもお願いします。

もしくは `LintResultResponse` を生成するタイミングで `status: str` を `status: LintStatus` に変更して `.value` で比較しなくても良いようにするのも良いかと思います